elements/lisk-chain/src/types.ts

```diff
@@ -107,3 +107,9 @@ export interface Validator {
 	minActiveHeight: number;
 	isConsensusParticipant: boolean;
 }
+
+export interface GenesisInfo {
+	height: number;
+	timestamp: number;
```

Comment:
are we using timestamp somewhere? i only saw height and initRounds usage, if not used maybe remove?